djangochat 2(Done)/room/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from .models import Room
+
+admin.site.register(Room)
```

Comment:
We could also register models to admin using the registerr decorator function. have a look at the docs and implement register decorator on the models to add them to admin instead.